features/step_definitions/networking.rb

```diff
@@ -1275,3 +1276,10 @@
   cb[cb_name]="172.31.249.80:9095"
   logger.info "The ipecho service url #{cb[cb_name]} is stored to the #{cb_name} clipboard."
 end
+
+Given /^the IPSec is enabled on the cluster$/ do
+  ensure_admin_tagged
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.ovnKubernetesConfig}")
+  raise "env doesn't have IPSec enabled" unless @result[:response].include? "ipsecConfig"
```

Comment:
Is there a reason for returning "ipsecConfig": {} instead of returning a boolean value in JSON for ipsec enabled OVN network?

Reply:
IPSec is only on OVN, yes. IMO its good to check ```ipsecConfig``` only as this is what is being relayed to customers via openshift 4.7 docs. Like if this is present in operator config, your cluster is supposed to be an IPSec cluster. Also it would be helpful in debugging in future looking at auto logs.